R/calc_aggregate_counts.R

```diff
@@ -29,6 +29,28 @@
 #' calc_aggregate_counts(state = TRUE, all_dates = TRUE)
 #' @export
 
+sub_vax_data <- function(df, ad){
+  grp_vars <- if(ad) c('State', 'Date') else c('State')
+  arrange_vars <- if(ad) c('State', 'Date', 'Measure') else c('State', 'Measure')
+  res <- df %>%
+    group_by_at(grp_vars) %>%
+    mutate(No.Initiated = !("Residents.Initiated" %in% Measure)) %>%
+    filter(No.Initiated ) %>%
+    filter(Measure %in% c("Residents.Completed")) %>%
+    arrange_at(arrange_vars) %>%
+    filter(1:n() == 1) %>% #if there are duplicates, take the highest one
```

Comment:
Sorry for being slow, but what is this filtering duplicates of?

Reply:
The way I read this, I think it would filter any out any overall duplicated rows (`1:n()`is not equal to 1) so that we only take one `Residents.Completed` value and transpose it onto a new `Residents.Initiated`. I don't see how this would necessarily take the 'highest' value though, or how a value would be highest in the event that we have a purely duplicated row. Maybe @lpw3 can answer that?

In my review, I couldn't find any scenario where there were actually duplicated rows. So one could argue that we should remove this line for clarity. I'd prefer to assume that it was added for some reason, and filtering out duplicates pro-actively is a good idea even if we don't need to.

Reply:
Yes, thanks Megan, that's correct! It arranges it by the highest value using the `arrange_at` function in the line above. I also didn't see any duplicate rows, but left it in in case there was some edge case that was added in for.